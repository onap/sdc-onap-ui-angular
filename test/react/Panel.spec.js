import React from 'react';
import Panel from '../../src/react/Panel.js';

import renderer from 'react-test-renderer';

describe('Panel', () => {
    test('Panel - Default', () => {
        const panel = renderer.create(<Panel>Panel</Panel>).toJSON();
        expect(panel).toMatchSnapshot();
    });
});
