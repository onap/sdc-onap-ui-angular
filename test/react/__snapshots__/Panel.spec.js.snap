// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel Panel - Default 1`] = `
<div
  className="sdc-panel "
>
  Panel
</div>
`;
